include/z64save.h

```diff
@@ -252,6 +252,16 @@ typedef enum {
     /* 3 */ GAMEMODE_END_CREDITS
 } GameMode;
 
+typedef enum {
+    /* 0 */ SCENE_LAYER_CHILD_DAY,
+    /* 1 */ SCENE_LAYER_CHILD_NIGHT,
+    /* 2 */ SCENE_LAYER_ADULT_DAY,
+    /* 3 */ SCENE_LAYER_ADULT_NIGHT    
+} SceneLayer;
+
+#define SCENE_LAYER_CUTSCENE_START (SCENE_LAYER_ADULT_NIGHT + 1)
+#define IS_CUTSCENE_LAYER (gSaveContext.sceneLayer >= SCENE_LAYER_CUTSCENE_START)
```

Comment:
looking through, it seems that more often than not you're negating `IS_CUTSCENE_LAYER`, which makes me think it should be expressed as the inverse instead.

Reply:
there are 46 `IS_CUTSCENE_LAYER` and 37 `!IS_CUTSCENE_LAYER`.

Reply:
never mind to my previous comment, did the search wrong.
IS_NIGHT is also more commonly negated, but we keep it in that form. I think it makes more sense to keep it as the positive rather than the negative